### PR TITLE
Change externalElasticsearchUrl to externalElasticsearchHost

### PR DESCRIPTION
**Pull request by mangoGoForward**
### What this PR dose?
Change externalElasticsearchUrl to externalElasticsearchHost at cc, and make the definition more clear.

### What this PR does / why we need it:
please see kubesphere/kubesphere#3829.

### Which issue(s) this PR fixes:

Fixes kubesphere/kubesphere#3829.

### CODE REVIEW

File: content/en/docs/pluggable-components/auditing-logs.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ By default, KubeKey will install Elasticsearch internally if Auditing is enabled
       elasticsearchDataVolumeSize: 20Gi    # The volume size of Elasticsearch data nodes.
       logMaxAge: 7                     # Log retention day in built-in Elasticsearch. It is 7 days by default.
       elkPrefix: logstash              # The string making up index names. The index name will be formatted as ks-<elk_prefix>-log.
-      externalElasticsearchUrl: # The URL of external Elasticsearch.
+      externalElasticsearchHost: # The URL of external Elasticsearch.
```
Comment (wenchajun) [inline content/en/docs/pluggable-components/auditing-logs.md:48]: externalElasticsearchHost: # The Host of external Elasticsearch.
is better

Comment (mangoGoForward) [reply content/en/docs/pluggable-components/auditing-logs.md]: Done.

**Comment by wenchajun** [comment]
/assign @Felixnoo 
Please take a look, thanks.

**Comment by Felixnoo** [comment]
/assign @wenchajun @pixiake 

Hi, I just found out that the changes in this PR are related to [this PR](https://github.com/kubesphere/helm-charts/pull/185#issuecomment-962969985). 

It seems that the released ks-installer v3.2.0 does not have the changes merged. Therefore, I am thinking that it is not the right time to merge the changes in this PR, which makes the documents inconsistent with the actual ks-installer v3.2.0 release.

Can you help confirm whether this PR should be merged?

**Pull request merged**